Resources/Prototypes/Roles/Antags/thief.yml

```diff
@@ -8,6 +8,8 @@
   requirements:
   - !type:OverallPlaytimeRequirement
     time: 84600 # DeltaV - was 3600, 24 hours
+  requirements:
+  - !type:WhitelistRequirement # DeltaV - Whitelist requirement
```

Comment:
it already has requirements change it to begin/end and add it to the existing one